support/mender-inventory-rootfs-type

```diff
@@ -2,5 +2,5 @@
 
 # Determines what the root filesystem type is.
 
-FS_TYPE="$(grep ' / ' /proc/mounts | awk '{print $3}')"
+FS_TYPE="$(grep ' / ' /proc/mounts | grep -v "^rootfs" | awk '{print $3}')"
```

Comment:
This will break detection if `rootfs` is all there is, in which case it is better to report `rootfs` than nothing. I suggest this solution instead:

```
FS_TYPE="$(grep ' / ' /proc/mounts | sort | tail -n 1 | awk '{print $3}')"
```

which is also imperfect (it will return `rootfs` if the real device sorts before `rootfs`), but at least it will cover your issue, as well as the issue when `rootfs` is by itself. And the name `ubi*` should be a pretty safe bet.

Reply:
I see your point, but I would argue that reporting `rootfs_type=rootfs` doesn't say anything. Maybe it's better to add
```bash
if [ -z "${FS_TYPE}" ]; then
    FS_TYPE=Unknown
fi
```
instead?

Reply:
Even better!